legal-api/src/legal_api/services/authz.py

```diff
@@ -267,6 +278,17 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
     },
     'general': {
         Business.State.ACTIVE: {
+            'agmLocationChange': {
+                'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
+                'blockerChecks': {
+                    'business': [
+                        BusinessBlocker.DEFAULT,
+                        BusinessBlocker.BUSINESS_FROZEN,
+                        BusinessBlocker.DRAFT_PENDING,
+                        BusinessBlocker.NOT_IN_GOOD_STANDING
+                    ]
+                }
+            },
```

Comment:
Just checking, but we wanted to allow both staff and general, correct?

Reply:
I think so.  But can you confirm with Mihai?

Reply:
Confirmed with Mihai that both will be used.

Reply:
ok, thanks for checking!